<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialogLocate</class>
 <widget class="QDialog" name="TDialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>定位单元格并设置文字</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>列号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Column"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Column">
          <property name="text">
           <string>列增</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>行号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Row"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Row">
          <property name="text">
           <string>行增</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>文字</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>21</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_SetText">
       <property name="text">
        <string>设置文字</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Close">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>TDialogLocate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
